SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+  We support the LTS Java releases and the current Java release.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability:
+  Under the repository name, click Security. If you cannot see the "Security" tab, 
+select the dropdown menu, and then click Security. Click Report a Vulnerability to open the advisory form.
+Fill out the form and then Submit the report.
+
+Someone from the team will contact you ASAP about the issue.
```

Comment:
Markdown is recommended to end with a new line after the last character.

Reply:
There is a new line at the end of line 14